src/components/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -0,0 +1,86 @@
+import {MarkdownEditorView, useMarkdownEditor, wToolbarConfig} from '@gravity-ui/markdown-editor';
+import {mToolbarConfig} from '@gravity-ui/markdown-editor/_/bundle/config/markup.js';
```

Comment:
You can import `mToolbarConfig` from root export of `@gravity-ui/markdown-editor`

Reply:
thx `markupToolbarConfigs. mToolbarConfig`